### ISSUE TITLE
SearchField: Sync with design on colors / Small Bug

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#6047]
**Describe the bug**
The colors need refinement and the focus state moves the field.

**To Reproduce**
1. Go to http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=new&mode=light
2. Click on search
3. Notice that the button shuffles

Also for design what colors should we use as some are a bit hard to see.
http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=new&mode=light
http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=classic&mode=light
http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=new&mode=dark
http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=classic&mode=dark
http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=new&mode=contrast
http://localhost:4000/components/header/example-toolbar-flex-with-categories.html?theme=classic&mode=contrast

**Expected behavior**
Confirm and check colors

**Version**
- ids-enterprise: [e.g. v4.60]

**Comment by tmcconechy** [comment]
@kevinwhitedesign could you give some guidance on this one?

**Comment by kevinwhitedesign** [comment]
@tmcconechy having trouble reproducing the environment

**Comment by tmcconechy** [comment]
@kevinwhitedesign try again i updated the links to live ones

**Comment by kevinwhitedesign** [comment]
@tmcconechy first link works, 404s on the others

**Comment by kevinwhitedesign** [comment]
I think we can refer to these designs @tmcconechy for the category search in the header https://www.figma.com/file/gS5jIE14am8hYEvmtIAXLv/?node-id=10%3A449

**Comment by kevinwhitedesign** [comment]
cc @laurensteiger have we explored dark / high contrast themes for these search fields?
Also cc @elizabethhartley

**Comment by laurensteiger** [comment]
Kevin and I explored different modes for the searchfields. Figma file can be seen here. https://www.figma.com/file/gS5jIE14am8hYEvmtIAXLv/Search_Enhancements?node-id=209%3A2536

**Comment by tmcconechy** [comment]
Thanks @laurensteiger this is helps but two questions.

1) What if the search field is in the header not on the second level (M3 request this to save space like the example https://main-enterprise.demo.design.infor.com/components/header/example-toolbar-flex-with-categories.html ) 
2) In dark mode the background color is still azure on the header https://main-enterprise.demo.design.infor.com/components/header/example-toolbar-flex-with-categories.html?theme=new&mode=dark - (not slate). Should we change that default? As its going to be wierd with a white background in dark mode?

**Comment by laurensteiger** [comment]
Hey Tim. Added two more header variants. The m3 request and an additional search field variant with the search button. For the second question, yes we should make slate the default.

**Comment by kevinwhitedesign** [comment]
@tmcconechy I think the M3 request works, just need to truncate/overflow titles and actions if they don't fit. And as long as background is white on the searchfield like the designs show

**Comment by tmcconechy** [comment]
Yeah the only thing i wonder about is making slate the default now. As it has been not slate for 5+ years

**Comment by kevinwhitedesign** [comment]
@tmcconechy Oh I missed that, keep as-is with Azure & personalization, we just used slate to show "this is a dark UI example" but nothing more than that.

**Comment by ericangeles** [comment]
@tmcconechy @kevinwhitedesign, is this still under design phase? Or can we move forward with the development?

**Comment by tmcconechy** [comment]
Yes @ericangeles this is now ready. See specs: https://www.figma.com/file/gS5jIE14am8hYEvmtIAXLv/Search_Enhancements?node-id=209%3A2536

**Comment by janahintal** [comment]
Failing in https://4620-beta0-enterprise.demo.design.infor.com/components/header/example-toolbar-flex-with-categories.html

Steps: 
1. Open in Mobile device iOS 15 iPhone 13
2. Type in Search field 
3. Notice the button didn't went responsive 
![image](https://user-images.githubusercontent.com/45757016/161707834-59274418-0b16-4fc2-9bf8-877e7c64c5c7.png)

Also for https://4620-beta0-enterprise.demo.design.infor.com/components/header/test-flex-toolbar-horizontal, the toolbar color is hard to read
![image](https://user-images.githubusercontent.com/45757016/161708576-6416fdda-7f0c-46d2-a90d-535547570425.png)

**Comment by tmcconechy** [comment]
I fixed one part (the button color) but will leave the responsive issue for now.

**Comment by kevinwhitedesign** [comment]
Oof, how come the responsive issue is happening?

**Comment by tmcconechy** [comment]
The difference is this never worked - vs the colors which broke. So what i mean by "for now" is will have to schedule the responsive issue for later. Logged here: https://github.com/infor-design/enterprise/issues/6271

**Comment by tmcconechy** [comment]
This issue is solved on main https://main-enterprise.demo.design.infor.com/ as it was an example issue only 
Second part to be addressed later https://github.com/infor-design/enterprise/issues/6271

**Comment by janahintal** [comment]
QA Passed. Thanks!
![image](https://user-images.githubusercontent.com/45757016/162164585-f3eccd44-daf4-4b71-a988-904cdf54a8ab.png)
